### PR TITLE
Add new babel dialogs

### PR DESCRIPTION
**Pull request by matiascalegaris**

### CODE REVIEW

File: CODIGO/Group.bas [hunk 1]
```diff
@@ -117,23 +122,25 @@ End Sub
...
-                If MainTimer.Check(TimersIndex.CastSpell) Then
-                    Call WriteActionOnGroupFrame(GroupMembers(i).GroupId)
-                    Call FormParser.Parse_Form(GetGameplayForm)
-                    UsaLanzar = False
-                    UsingSkill = 0
-                    If CursoresGraficos = 0 Then
-                        GetGameplayForm.MousePointer = vbDefault
+    If Not Hide Then
+        For i = 0 To GroupSize - 1
+            If PointIsInsideRect(x, y, GroupMembers(i).RenderArea) Then
```
Comment (morgolock) [inline CODIGO/Group.bas:127]:
```With GroupMembers(i)
        For i = 0 To GroupSize - 1
            If PointIsInsideRect(x, y, .RenderArea) Then
```

File: CODIGO/Group.bas [hunk 2]
```diff
@@ -117,23 +122,25 @@ End Sub
...
+                If UsingSkill = magia Then
+                    HandleMouseInput = True
+                    If MainTimer.Check(TimersIndex.CastSpell) Then
+                        Call WriteActionOnGroupFrame(GroupMembers(i).GroupId)
```
Comment (morgolock) [inline CODIGO/Group.bas:131]: .GroupId

Comment (morgolock) [inline CODIGO/Group.bas:131]: .GroupId

File: CODIGO/ModGameplayUI.bas [hunk 3]
```diff
@@ -232,6 +233,29 @@ OnClick_Err:
     Resume Next
 End Sub
 
+Public Sub HandleQuestionResponse(ByVal Result As Boolean)
```
Comment (morgolock) [inline CODIGO/ModGameplayUI.bas:236]: Is this the callback that BABEL uses to notify the game the user choice?

**Review by morgolock — APPROVED** [review]

File: CODIGO/ModGameplayUI.bas (hunk 3 above)
Comment (matiascalegaris) [reply CODIGO/ModGameplayUI.bas:236]: no, this is the common point for both babel and legacy ui, its called from both babel and here in HandleGameplayAreaMouseUp

**Pull request merged**